docs/src/howto/async-commit-merge.md

```diff
@@ -0,0 +1,436 @@
+---
+title: Asynchronous Commit and Merge
+description: Perform large commit and merge operations asynchronously to avoid timeouts and indeterminate outcomes.
+---
+
+# Asynchronous Commit and Merge
+
+!!! info
+    Available in **lakeFS Enterprise** and **lakeFS Cloud** only.
+
+Asynchronous commit and merge handle large operations that may exceed timeout limits. Operations return immediately with an operation ID that you poll for status, ensuring reliable completion for commits and merges involving millions of objects.
+
+## When to Use
+
+Use async operations when:
+
+- Committing or merging millions of objects that exceed gateway timeout limits
```

Comment:
Do we know the current timeout limits?
Or is this different per installation?

Reply:
This will be different per installation, but this line could be clearer to indicate that.